### DISCUSSION TITLE
What about SAP HANA ?

### DISCUSSION THREAD

**Discussion by rferraton** [sfu-db/connector-x#272] [Ideas]
There is a start of rust driver for SAP HANA  : [hdbconnect](https://github.com/emabee/rust-hdbconnect )
It may help for a new database connector.

**Comment by wangxiaoying** [comment]
Hi @rferraton , thank you for sharing this info. It is helpful!

**Reply by rferraton** [reply]
fell free to ping us if you need help for test environment setup.  We (me with @djfrancesco) could help for testing (not programming unfortunately). Setting up and HANA Express is not as straight forward than a postgresql db). I could also test a future hana connect in a real world environment.
